Bdd/features/steps/syslog_steps.py

```
@@ -58,18 +62,46 @@ def parse_syslog_line(line):
+    process.stdin.flush()
+    return wait_for_prompt(process)
+
+
+def wait_for_messages(context, expected_messages):
+    """Wait for syslog-ng to flush the expected number of new lines."""
+    expected_total = context.lines_before + expected_messages
+    deadline = time.monotonic() + 5
+    while line_count(RECEIVED_LOG) < expected_total:
+        if time.monotonic() > deadline:
+            actual = line_count(RECEIVED_LOG) - context.lines_before
+            assert False, (
```

Ruff (0.15.9)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)